src/modules/AccordionPanelGrommet.js

```diff
@@ -0,0 +1,143 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import Anchor from 'grommet/components/Anchor';
+import Box from 'grommet/components/Box';
+import Heading from 'grommet/components/Heading';
+import ListItem from 'grommet/components/ListItem';
+import Paragraph from 'grommet/components/Paragraph';
+import Tile from 'grommet/components/Tile';
+import Tiles from 'grommet/components/Tiles';
+import OpenIcon from 'grommet/components/icons/base/Add';
+import CloseIcon from 'grommet/components/icons/base/Subtract';
+
+const CLASS_ROOT = 'accordion-panel-grommet';
+
+export default class AccordionPanelGrommet extends Component {
+  constructor (props) {
+    super(props);
+
+    this._renderCTA = this._renderCTA.bind(this);
+    this._renderResources = this._renderResources.bind(this);
+    this._onClickPanel = this._onClickPanel.bind(this);
+
+    this.state = {
+      isOpen: false
+    };
+  }
+
+  _renderCTA () {
+    let { ctaThumbnail } = this.props;
+    // let { ctaThumbnail, ctaVideo } = this.props;
+    let { headline, subHeadline, link, linkText } = this.props.ctaText;
+    // TODO: add video overlay functionality for ctaVideo
+
+    return (
+      <Box direction="row" pad={{vertical: "medium"}}>
+        <img src={ctaThumbnail} className="accordion-cta-image" />
```

Comment:
Can we access the `img` element using CSS from the parent instead of creating adding a new class?

Also, if we need to add a new class to an element, can you please use the BEM convention to naming your classes to match Grommet core?